packages/snackbar/test/mwc-snackbar.test.ts

```diff
@@ -57,8 +57,10 @@ suite('mwc-snackbar', () => {
 
   setup(() => {
     originalSetTimeout = window.setTimeout;
-    (window as any).setTimeout = (fn: Function) => {
-      fn();
+    window.setTimeout = (fn) => {
+      if (typeof fn === 'function') {
+        fn();
+      }
```

Comment:
this is what i ended up with @e111077. i hate the  condition here but we're in a predicament...

basically the linter is working against us in this case because `setTimeout` is typed as `fn: string|Function`. so if we annotate our types here or forcefully cast as `Function` we will hit the ESLint rule yet again (telling us off for using `Function` instead of a specific signature).

this seems to work fine but if we want to avoid having this condition we will have to ignore the lint rule. let me know what you think

Reply:
why not just cast fn in the setTimeout as a function in TS? e.g. `(fn as ()=>unknown)();`?